packages/babel-make-styles/src/utils/evaluatePathsInVM.ts

```diff
@@ -55,58 +54,24 @@ const expressionWrapperTpl = template.statement(`
   };
 `);
 
-/**
- * Functions, call & member expressions should be wrapped with IIFE to ensure that "theme" object will be passed
- * without collisions.
- *
- * @example
- * {
- *   label: foo(), // call expression
- *   header: typography.header, // could be an object or a function
- * }
- *
- * Outputs following template:
- * @example
- * wrap(() => typeof foo === 'function' ? foo(theme) : foo)
- */
-export const expressionTpl = template.expression(
-  `%%wrapName%%(() => typeof %%expression%% === 'function' ? %%expression%%(%%themeVariableName%%) : %%expression%%)`,
-);
+export const expressionTpl = template.expression(`%%wrapName%%(() => %%expression%%)`);
 const exportsPrevalTpl = template.statement(`exports.${EVAL_EXPORT_NAME} = %%expressions%%`);
 
 /**
  * Creates a new program that includes required imports and wrappers to return resolved values.
  */
-function addPreval(
-  path: NodePath<t.Program>,
-  themeVariableName: string,
-  lazyDeps: Array<t.Expression | t.SpreadElement>,
-): t.Program {
-  // Constant __mkPreval with all dependencies
+function addPreval(path: NodePath<t.Program>, lazyDeps: Array<t.Expression | t.SpreadElement>): t.Program {
+  // Constant for "__mkPreval" with all dependencies
   const wrapName = findFreeName(path.scope, '_wrap');
-  const proxyImportName = path.scope.generateUid('createCSSVariablesProxy');
-
   const programNode = path.node;
 
   return t.program(
-    // Temporary solution to solve "theme" dependency
     [
-      t.importDeclaration(
-        [t.importSpecifier(t.identifier(proxyImportName), t.identifier('createCSSVariablesProxy'))],
-        t.stringLiteral('@fluentui/make-styles'),
-      ),
-
-      t.variableDeclaration('const', [
-        t.variableDeclarator(t.identifier(themeVariableName), t.callExpression(t.identifier(proxyImportName), [])),
-      ]),
```

Comment:
As we don't have builtin theming (#21239) we don't need to import these anymore.